book/website/reproducible-research/open/open-resources.md

```diff
@@ -1,33 +1,6 @@
 (rr-open-resources)=
 # Resources
 
-(rr-open-resources-checklist)=
-## Checklist
-
-## Open source software
```

Comment:
I don't see these section headers are being duplicated anywhere. Or do you mean the info is duplicated rather than a verbatim duplicate?

Reply:
No the checklist is available elsewhere: https://github.com/alan-turing-institute/the-turing-way/blob/e85a6fac244d88458b6effe2e60a68b0f32fd28a/book/website/reproducible-research/open/open-checklist.md

Reply:
> The rdm-personal sections are indeed duplicates, but I couldn't find the other ones. Could you provide some more info about those duplicates?

Was the checklist the only one? Or am I missing something? Please let me know and thanks for reviewing!